bee-node/src/cli.rs

```diff
@@ -45,7 +40,15 @@ struct Cli {
     commit_version: bool,
     // Whether the node should run as an (autopeering) entry node.
     #[structopt(long = "entry_node", help = "Runs as autopeering entry node")]
-    entry_node: bool,
+    run_as_entry_node: bool,
+    // Whether the node should run with enabled autopeering service.
+    #[structopt(long = "autopeering", help = "Enables the autopeering service")]
+    enable_autopeering: bool,
+}
+
+/// Holds the command line arguments that were passed to the binary.
+pub struct ClArgs {
```

Comment:
What's the point of this struct btw ? Why don't we have only one ?

Reply:
See what I wrote in the BEWARE comment. Basically `structopt` structs turn doc comments (intended for developers) into user facing help text. I wanted to have this under our control, that's why the the wrapper struct.